qa/acceptance/spec/shared_examples/cli/logstash-plugin/install.rb

```diff
@@ -0,0 +1,59 @@
+# encoding: utf-8
+require_relative "../../../spec_helper"
+require "logstash/version"
+require "fileutils"
+require "logstash/devutils/rake"
+
+shared_examples "logstash install" do |logstash|
+  before(:each) do
+    logstash.install(LOGSTASH_VERSION)
+  end
+
+  after(:each) do
+    logstash.uninstall
+  end
+
+  describe "on #{name}" do
```

Comment:
we should use here logstash.hostname instead of name, name is a custom variable created by rspec, so what it prints is actually this.

```
  behaves like logstash install
    on RSpec::ExampleGroups::CLIOperation::BehavesLikeLogstashInstall
      with a direct internet connection
```

see https://github.com/elastic/logstash/pull/5323 as example.